flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/Checklist/Library.xml

```diff
@@ -99,7 +99,7 @@
          <Instrument Id="LANDING_GEAR_Switch_ParkingBrake" />
          <Test>
             <TestValue>
-               <Val SimVarName="BRAKE PARKING POSITION" Units="Boolean"/>
+               <Val SimVarName="L:A32NX_PARK_BRAKE_LEVER_POS" Units="Boolean"/>
```

Comment:
Please add the simvar to `a320-simvars.md`.

Reply:
Think I missed totally that file for that PR!